packages/sdk-components-react/src/head-link.ws.ts

```diff
@@ -0,0 +1,22 @@
+import { Link2Icon } from "@webstudio-is/icons/svg";
```

Comment:
maybe combine into single module like radix components?

Reply:
yep will do

Reply:
Decided to not. Easier as now